plugin-flex-ts-template-v2/src/feature-library/custom-transfer-directory/types/ServiceConfiguration.ts

```diff
@@ -1,3 +1,5 @@
+import { ExternalDirectoryEntry } from './DirectoryEntry';
+
 export default interface CustomTransferDirectoryConfig {
   enabled: boolean;
```

Comment:
i think this is missing the new worker config definition

Reply:
Fixed!